lib/server/swarm.js

```diff
@@ -7,12 +7,27 @@ var randomIterate = require('random-iterate')
 // Regard this as the default implementation of an interface that you
 // need to support when overriding Server.createSwarm() and Server.getSwarm()
 function Swarm (infoHash, server) {
+  this.complete = 0
+  this.incomplete = 0
   this.peers = new LRU({
     max: server.peersCacheLength || 1000,
-    maxAge: server.peersCacheTtl || 900000 // 900 000ms = 15 minutes
+    maxAge: server.peersCacheTtl || 20 * 60 * 1000 // 20 minutes
```

Comment:
This value is used for the udp and http trackers too.

I took a look at the source of opentracker (the most popular tracker software) and noticed they use 30 minutes on the server, but they tell clients to announce every 15 minutes by default.

We're currently telling clients to announce every 15 minutes and purging them at the same time. Which means if they're only a few seconds late, they'll already be removed, and they'll need to be re-added to the LRU. Better to give them some leeway.

For websocket connections, I think this value probably matters less. If the client closes the websocket, they are removed immediately (now that we added the 'evict' handling). So, the only possible downside of a large value for a websocket tracker is if someone opens a websocket but doesn't send any data on it for a long time. The server will keep the connection open for 20 minutes now. But no well-behaved clients should do that, right?

The LRU is shared for all tracker types, so we need to keep it > 15 min.